src/platform/packages/shared/kbn-esql-ast/src/sorting/priority_provider.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import type { ISuggestionItem } from '../commands_registry/types';
+import type { SortingContext } from './types';
+import { SuggestionCategory } from './types';
+import { detectCategory } from './utils/category_rules';
+
+const CATEGORY_PRIORITIES: Record<SuggestionCategory, number> = {
+  [SuggestionCategory.CRITICAL_ACTION]: 0,
+  [SuggestionCategory.KEYWORD_CLAUSE]: 50, // BY, WHERE, ON, WITH
+
+  [SuggestionCategory.TIME_PARAM]: 100,
+
+  [SuggestionCategory.COMMA]: 200,
+  [SuggestionCategory.PIPE]: 200,
+  [SuggestionCategory.KEYWORD]: 200,
```

Comment:
```suggestion
  [SuggestionCategory.VALUE]: 200,
```

Are these values suggestions? If yes, value is a better name, otherwise explain to me to find a better name. Keyword is so generic that this naming here is very confusing